src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form search-form_big"
+      action="/"
+      method="get"
+      data-qa="big"
+      >
+      <label class="search-form__label search-form__label_big">
+        <input
+        data-qa="keypress"
+        type="search"
+        class="search-form__input search-form__input_big"
+        id="search-input"
+        placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
```

Comment:
Pay attention on indents. Fix same with small input
```suggestion
    <form
      class="search-form search-form_big"
      action="/"
      method="get"
      data-qa="big"
    >
      <label class="search-form__label search-form__label_big">
        <input
          data-qa="keypress"
          type="search"
          class="search-form__input search-form__input_big"
          id="search-input"
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
    </form>
```